app/assets/javascripts/filters.js

```diff
@@ -0,0 +1,70 @@
+var initFilters = function() {
+  var reload = function(url) {
+    $('#search-loader__panel').show();
+    window.history.pushState({}, "", url);
+    $.ajax({
+      url: url,
+      type: 'GET',
+      dataType: 'script'
+    });
+  };
+
+  $('input[name="daterange"]').daterangepicker({
+    "startDate": moment().subtract(7, 'days'),
+    "endDate": moment(),
+    "buttonClasses": "btn btn-sm btn-gc",
+    locale: {
+      format: 'DD/MM/YYYY'
```

Comment:
Can you change the format to the American format? `MM/DD/YYYY`. Only because the initial audience will be from the US :)